Files/DataModels/FilesystemItemsOperationDataModel.cs

```diff
@@ -62,7 +67,15 @@ public List<FilesystemOperationItemViewModel> ToItems(Action updatePrimaryButton
             // Add conflicting items first
             foreach (var item in ConflictingItems)
             {
-                items.Add(new FilesystemOperationItemViewModel(updatePrimaryButtonEnabled, optionGenerateNewName, optionReplaceExisting, optionSkip)
+                var (IconData, OverlayData, IsCustom) = await associatedInstance.FilesystemViewModel.LoadIconOverlayAsync(item.SourcePath, 64u);
```

Comment:
In #5112 I'm removing this method, you can use the identical `FileThumbnailHelper.LoadIconOverlayAsync()`. This way you may not need to pass the `associatedInstance` anymore.